hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/utils/DiskCheckUtil.java

```diff
@@ -41,6 +41,11 @@ private DiskCheckUtil() { }
   // to inject failures.
   private static DiskChecks impl = new DiskChecksImpl();
 
+  /** Enum for disk read/write status during volume check. */
+  public enum ReadWriteStatus {
+    WRITE_FAIL, READ_FAIL, READ_WRITE_OK
+  }
```

Comment:
This naming is more consistent with the container scanner, which uses [ScanResult](https://github.com/apache/ozone/blob/d7e15f5733b997bca60d1921192d7497c6500a37/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/Container.java#L44). FWIW we moved the scan result to a [separate class](https://github.com/apache/ozone/blob/445eaf1f4a8748d0187629d65eca61ba93bc18f0/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/ScanResult.java#L27) in the reconciliation branch. Volume scan is simpler than container scan though so I think enum is good for now.
```suggestion
  public enum DiskCheckResult {
    WRITE_FAILURE, READ_FAILURE, OK
  }
```